01-introduction.Rmd

```diff
@@ -149,7 +151,7 @@ In fact, we often advise our students to start with Python just because the majo
 ].
 However, when the teaching moves on to statistical geoprocessing and spatial predictive modeling we guide them towards R where they can take advantage of the concepts already learned through Python.
 Nevertheless, you can also use Python for the most common statistical learning techniques (though R tends to be more on the bleeding edge regarding new statistical development including those in the geostatistical community).
-In addition, Python also offers an excellent support for dealing with spatial data (see packages **osgeo**, **Shapely**, **NumPy**, **osgeo**, **PyGeoProcessing**). 
+In addition, Python also offers excellent support for spatial data analysis and manipulation (see packages **osgeo**, **Shapely**, **NumPy**, **osgeo**, **PyGeoProcessing**). 
```

Comment:
Great that you've got so specific. I've just been looking at **geopandas** but not sure it's worth referencing as it does not seem to be well-maintained. https://github.com/geopandas/geopandas